include/wx/arrstr.h

```diff
@@ -81,6 +84,8 @@ class WXDLLIMPEXP_BASE wxArrayString : public wxArrayStringBase
     wxArrayString(size_t sz, const char** a);
     wxArrayString(size_t sz, const wchar_t** a);
     wxArrayString(size_t sz, const wxString* a);
+    template<typename _IList>
```

Comment:
We probably should stick to using `T` for the generic template parameters, especially because `_IList` is a bit confusing (it's the type of the initializer list elements and not of the list itself) and is also a reserved identifier.

Reply:
OK, since `T` is the name of the template parameter in some other definitions I opted for a more generic name while being consistent all over the place. I can change that to `U` I guess.
Do you have any documentation about `_IList` being a reserved identifier?

Reply:
From https://en.cppreference.com/w/cpp/language/identifiers (which is authoritative enough, but the Standard has the same wording):

Identifiers that appear as appearing as a token or preprocessing token of one of the following forms are reserved:

- identifiers with a double underscore anywhere;
- identifiers that begin with an underscore followed by an uppercase letter;
- in the global namespace, identifiers that begin with an underscore. 

And `_IList` falls in the second category, of course.

Reply:
Thanks a lot for the details, I was not aware. Very useful to know.
This is fixed and I have replaced `_IList` with `Ty`